models.py

```diff
@@ -39,6 +58,16 @@ class DatasetComputedColumns:
         {"field": "contact_points", "exclude": DEFAULT_LIST_EXCLUDE},
     ]
 
+    @staticmethod
+    def get_bin(value, bounds) -> Bin:
+        bin = len(list(takewhile(lambda v: v <= value, bounds.values)))
```

Comment:
You might be interested in https://docs.python.org/3/library/bisect.html#bisect.bisect_right